app/pages/sections/order-items/catalogue-solutions/order-item/flat/patient/ui-tests/withSavedData.ui.test.js

```diff
@@ -0,0 +1,294 @@
+import nock from 'nock';
+import { ClientFunction, Selector } from 'testcafe';
+import { extractInnerText } from 'buying-catalogue-library';
+import { orderApiUrl } from '../../../../../../../../config';
+import content from '../manifest.json';
+import { nockAndErrorCheck, setState, authTokenInSession } from '../../../../../../../../test-utils/uiTestHelper';
+import { sessionKeys } from '../../../../../../../../helpers/routes/sessionHelper';
+
+const pageUrl = 'http://localhost:1234/order/organisation/order-1/catalogue-solutions/item-1';
+
+// const getLocation = ClientFunction(() => document.location.href);
```

Comment:
Is this the one where you wanted to retain the commented-out code?

Reply:
yeah it is